pypeit/core/coadd.py

```diff
@@ -1842,122 +1939,76 @@ def spec_reject_comb(wave_grid, waves, fluxes, ivars, masks, weights, sn_clip=30
     return wave_stack, flux_stack, ivar_stack, mask_stack, outmask, nused
 
 
-def scale_spec_stack(wave_grid, waves, fluxes, ivars, masks, sn, weights, ref_percentile=70.0, maxiter_scale=5,
-                     sigrej_scale=3.0, scale_method='auto', hand_scale=None, sn_min_polyscale=2.0, sn_min_medscale=0.5,
+def scale_spec_stack(wave_grid, waves, fluxes, ivars, masks, sn, weights, 
+                     ref_percentile=70.0, maxiter_scale=5,
+                     sigrej_scale=3.0, scale_method='auto', 
+                     hand_scale=None, sn_min_polyscale=2.0, sn_min_medscale=0.5,
                      debug=False, show=False):
 
     '''
-    Routine for optimally combining long or multi-slit spectra or echelle spectra of individual orders. It will
-    compute a stacked spectrum from a set of exposures on the specified wave_grid with proper treatment of
-    weights and masking. This code calls the stacking code compute_stack, which uses np.histogram to combine the data using
-    NGP and does not perform any interpolations and thus does not correlate errors. It uses wave_grid to determine the set
-    of wavelength bins that the data are averaged on. The final spectrum will be on an ouptut wavelength grid which is not
-    the same as wave_grid. The ouput wavelength grid is the weighted average of the individual wavelengths used for each
-    exposure that fell into a given wavelength bin in the input wave_grid. This 1d coadding routine thus maintains the
-    independence of the errors for each pixel in the combined spectrum and computes the weighted averaged wavelengths of
-    each pixel in an analogous way to the 2d extraction procedure which also never interpolates to avoid correlating
-    erorrs. It performs a number of iterations where it combines the spectra and performs rejection of outlier pixels
-    using the spec_reject_comb code. The outliers are rejected using the true noise of the individual exposures, but
-    uses the distribution of the pixel values about the stack to apply correction factors to the errors before rejecting.
-    These corrected errors are currently only used in rejection but are not applied to the data.  This code is based
-    on the xidl long_combpsec.pro routine but with significant improvements.
+    THIS NEEDS A PROPER DESCRIPTION
```

Comment:
Was there something wrong with the deleted description?

Reply:
yeah, it may have been generically correct but was not informative.